### ISSUE TITLE
Preview opens only once

### ISSUE THREAD

**Issue by dsentker** [etimbo/jquery-print-preview-plugin#1]
The fancy preview overlay opens only once when i click on "print this page". When i click on the body background, the overlay will close and the "print this page" link do not trigger.

Sorry for my bad english.

**Comment by Guite** [comment]
It would be very cool if this issue could be addressed. Thanks for this nice script.

**Comment by witiokz** [comment]
This issue appears in all versions of Firefox (or maybe in 7 only). So the temporary working solution could be this:

after line 44 var print_frame_ref = window.frames["print-frame"].document;  paste this:

```
                    if(print_frame_ref == null) {
            for(var i=0; i < window.frames.length; i++) {
                if(window.frames[i].name == "print-frame") {    
                    print_frame_ref = window.frames[i].document;
                    break;
                }
            }
        }
```

**Comment by etimbo** [comment]
Thanks for reporting this issue.
Resolved as of v1.0.1.
